interface/storage/pools-btrfs.rst

```diff
@@ -31,66 +31,144 @@ There is a tooltip for each input field to help you choose appropriate parameter
 Redundancy profiles
 ^^^^^^^^^^^^^^^^^^^
 
-All standard BTRFS redundancy profiles are available when creating a pool.
+A broad abstraction of BTRFS raid profiles are available when creating a pool.
+
+- Rockstor 4.1.0-0 and earlier supported btrfs raid single, 0, 1, 10, 5, and 6; with no mixed profile awareness.
+- 4.6.0-0's Web-UI supports: single, single-dup, 0, 1, 10, 5, 6, raid1c3, raid1c4, raid1-1c3, raid1-1c4,
```

Comment:
```suggestion
- Starting with Rockstor 4.6.0-0 the Web-UI now supports: single, single-dup, 0, 1, 10, 5, 6, raid1c3, raid1c4, raid1-1c3, raid1-1c4,

Reply:
@Hooverdan96 I had intentionally not repeated the Subject (Our project) but lets go with this :). Better when abstracted as a quote with your suggestion.